assets/styles/layout/_footer.scss

```
@@ -41,101 +60,154 @@
   }
   &__contact {
     &--detail {
-      margin-bottom: 66px;
+      margin-bottom: 6.6rem;
 
       &:nth-child(2) {
-        margin-bottom: 33px;
+        margin-bottom: 7.2rem;
       }
     }
     &--items {
       display: flex;
       flex-direction: row;
       flex-wrap: wrap;
+      width: 470px;
+      margin-inline: auto;
+
+      @include mq(lg) {
+        width: 100%;
+        margin-left: 0;
+      }
+
       @media screen and (max-width: 1150px) {
         display: block;
+        width: fit-content;
       }
     }
 
     &--item {
       width: 50%;
+      margin-bottom: 17px;
 
       @media screen and (max-width: 1150px) {
         width: 100%;
       }
     }
     &--itemFlex {
       display: flex;
-      gap: 22px;
+
+      // 650以下になるとブラウザの表示より幅大きくなるための処理
+      @media screen and (max-width: 650px) {
+        display: block;
+      }
+
+      @include mq(lg) {
+        gap: 22px;
+      }
     }
     &--title {
-      color: #fff;
+      color: #e2e9f1;
+      text-align: center;
       font-family: Noto Sans CJK JP;
+      font-size: 2rem;
+      font-style: normal;
+      font-weight: 700;
+      line-height: 150%;
```

line-heightは1.5のように書くことが多いので、そちらでお願いします！

多分下記で同義なはず
```suggestion
      line-height: 1.5;
```

---

次のコミットで全体のline-heghtを上記の形式に修正します！